esx_service/vmdk_ops_test.py

```diff
@@ -1201,6 +1175,211 @@ def test_vmdkops_for_default_datastore(self):
+                                           name=auth.DEFAULT_TENANT,
+                                           description="This is a default tenant",
+                                           vm_list=[],
+                                           privileges=[])
+        if error_info:
+            err = error_code.TENANT_CREATE_FAILED.format(auth.DEFAULT_TENANT, error_info)
+            logging.warning(err)
+
+        # create DEFAULT privilege if needed
+        error_info, privileges = auth.get_default_privileges()
+        if not privileges:
+            logging.debug("create_default_tenant_and_privileges: create DEFAULT privileges")
+            error_info = auth_api._tenant_access_add(name=auth.DEFAULT_TENANT,
+                                                    datastore=auth.DEFAULT_DS,
+                                                    allow_create=True,
+                                                    default_datastore=False,
+                                                    volume_maxsize_in_MB=0,
+                                                    volume_totalsize_in_MB=0)
+            if error_info:
+                err = error_code.TENANT_SET_ACCESS_PRIVILEGES_FAILED.format(auth.DEFAULT_TENANT, auth.DEFAULT_DS, error_info)
+                logging.warning(err)
+
+    def setUp(self):
+        """ Setup run before each test """
+        logging.info("VmdkTenantPolicyUsageTestCase setUp path =%s", path)
+
+        if (not self.datastore_name):
+            datastores = vmdk_utils.get_datastores()
+            if datastores:
+                datastore = datastores[0]
+                self.datastore_name = datastore[0]
+                self.datastore_path = datastore[2]
+                logging.debug("datastore_name=%s datastore_path=%s", self.datastore_name,
+                                                                     self.datastore_path)
+                if len(datastores) > 1:
+                    datastore1 = datastores[1]
+                    self.datastore1_name = datastore1[0]
+                    self.datastoer1_path = datastore[2]
+                    logging.debug("Found second datastore: datastore_name=%s datastore_path=%s",
+                                  self.datastore1_name, self.datastore1_path)
+            else:
+                logging.error("Cannot find a valid datastore")
+                self.assertFalse(True)
+
+        self.cleanup()
+        # get service_instance, and create VMs
+        si = vmdk_ops.get_si()
+        error, self.vm1 = create_vm(si=si,
+                                    vm_name=self.vm1_name,
+                                    datastore_name=self.datastore_name)
+        if error:
+            self.assertFalse(True)
+
+        self.vm1_config_path = vmdk_utils.get_vm_config_path(self.vm1_name)
+        logging.info("VmdkTenantPolicyUsageTestCase: create vm1 name=%s Done", self.vm1_name)
+
+        error, self.vm2 = create_vm(si=si,
```

Comment:
Is there a reason for creating second vm? it seems your test is not consuming second vm unless I have mistaken. Please clarify here.